falcon/testing/client.py

```diff
@@ -371,6 +377,10 @@ def simulate_request(app, method='GET', path='/', query_string=None,
             prefix=False,
         )
 
+    if content_type is not None:
```

Comment:
`if content_type is not None` -> `if content_type`?

Reply:
The caller probably won't ever need to set the header to an empty string, but it may make sense to allow it just in case.

Reply:
Understood, in that case, it make sense 👍